desc/equilibrium/equilibrium.py

```diff
@@ -2324,8 +2325,7 @@ def perturb(
         ----------
         objective : ObjectiveFunction or LinearConstraintProjection
             Objective function to satisfy. Default = force balance. If
-            LinearConstraintProjection is used, no constraints are needed. This is
-            useful for eq update in ProximalProjection.
+            LinearConstraintProjection is used, no constraints are needed.
```

Comment:
see above. Basically I don't think these docstrings should change at all, since from a user standpoint the functions/methods are exactly the same, we're just adding some private stuff behind the scenes. You can move it to a commend in the function so other devs know what's going on.